pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -348,13 +349,93 @@ func (p *staticPolicy) GetTopologyHints(s state.State, pod *v1.Pod, container *v
 
 	// Generate hints.
 	cpuHints := p.generateCPUTopologyHints(available, reusable, requested)
-	klog.Infof("[cpumanager] TopologyHints generated for pod '%v', container '%v': %v", pod.Name, container.Name, cpuHints)
+	klog.Infof("[cpumanager] TopologyHints generated for pod '%v', container '%v': %v", format.Pod(pod), container.Name, cpuHints)
 
 	return map[string][]topologymanager.TopologyHint{
 		string(v1.ResourceCPU): cpuHints,
 	}
 }
 
+func (p *staticPolicy) getPodRequestedCPU(pod *v1.Pod) int {
```

Comment:
nit: can you rename this to `podGuaranteedCPUs()` instead of `getPodRequestedCPU()` and move it just underneath the defintion of the `guaranteedCPUs()` function.

It is analogous to the existing function `guaranteedCPUs()` (which works on containers), so this new name makes it clearer that they are related (when looking at it in isolation).

This also makes it more consistent with with:
```
GetTopologyHints() --> GetPodTopologyHints()
guaranteedCPUs() --> podGuaranteedCPUs()
```

Reply:
It doesn't appear that this was done.

Reply:
Checked and ensured it is done.